### ISSUE TITLE
Nextcloud Collabora file versions

### ISSUE THREAD

**Issue by mpiscaer** [nextcloud/server#4172]
Hi,

When I create an Collabora file and online edit it and after that close the file.

Nextcloud doesn't create a version file. Also if edit it again it also doesn't create a new version of it. If I after edit the file offline. The nextcloud does create a new version of the file.

Nextcloud is on version 11.0.2 and I last week I did an docker pull Collabora command. So both are recent.

Also I created an screencast for this: https://share.piscaer.com/index.php/s/F4BmIyZU5GOWyZU The password is: This_is_the_public_passw0rd

**Comment by MorrisJobke** [comment]
Could you try https://github.com/nextcloud/richdocuments/pull/27 and https://github.com/nextcloud/richdocuments/pull/33 ? Those two together should fix that issue.

**Comment by mpiscaer** [comment]
Thanks,

I applied both and this fix the versioning problem.

I created test2.odt and then I edited the file two times. And now I see two versions, like you see on the screenshot:

![screenshot from 2017-04-04 09 56 19](https://cloud.githubusercontent.com/assets/1213043/24646816/7eafd3fc-191d-11e7-8d2a-5778d0239af5.png)
 
But on the activity my name doesn't get mention that I change the file. It onlye gives you "Changed by ..."
![screenshot from 2017-04-04 09 56 29](https://cloud.githubusercontent.com/assets/1213043/24646884/d37ddea6-191d-11e7-98c1-bfe6514c586d.png)
